### ISSUE TITLE
Tooltip Improvements

### ISSUE THREAD

**Issue by hk21702** [RimSort/RimSort#627]
### Origin of idea

Currently, the vast majority of buttons, options, etc., lack static tooltips. As such, there is often confusion that can be easily cleared up such as the difference between deletion and unsubscribing.

### Suggestion/proposed solution?

Add static tooltips with relevant, complete, yet brief information to UI elements where applicable.

- [ ] Mod context menu
- [ ] Settings
- [ ] Mod sorting
- [ ] Rules editor
- [ ] Application menu options

### Code of Conduct

I agree to follow this project's Code of Conduct

### Duplicate Issue Check

I've checked for similar issues and didn't find anything

### Wiki/FAQ Check

I've checked the Wiki for a solution and didn't find a solution
